packages/repl/src/lib/workers/bundler/index.js

```diff
@@ -286,8 +286,7 @@ async function get_bundle(uid, mode, cache, local_files_lookup) {
 				if (importer && local_files_lookup.has(importer)) {
 					// relative import in a REPL file
 					// should've matched above otherwise importee doesn't exist
-					console.error(`Cannot find file "${importee}" imported by "${importer}" in the REPL`);
-					return;
+					throw new Error(`Cannot find file "${importee}" imported by "${importer}" in the REPL`);
```

Comment:
I remember I did it because the compiler was hanging fully on any error and refusing to re-run on typing. lemme check if this works